README.md

```diff
@@ -20,6 +22,18 @@ snap install signal-desktop
 
 <p align="center">Published for <img src="https://raw.githubusercontent.com/anythingcodes/slack-emoji-for-techies/gh-pages/emoji/tux.png" align="top" width="24" /> with :gift_heart: by Snapcrafters</p>
 
+### Building snap locally
+
+``` bash
+sudo snap install --classic snapcraft
+git clone https://github.com/snapcrafters/signal-desktop.git
+cd signal-desktop
+snapcraft
+sudo snap install --devmode signal-desktop_*.snap
```

Comment:
this should be `--dangerous` instead of `--devmode`. Otherwise, you turn off the sandbox.